install_config/install/prerequisites.adoc

```diff
@@ -362,6 +362,121 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[[prereq-cloud-provider-considerations]]
+
+=== Cloud Provider Considerations
+
+*Set up the Security Group*
+
+When installing on AWS or OpenStack, ensure that you set up the security group.
+
+[cols="1,2"]
+|===
+|*All OpenShift Hosts*
+a|tcp/22 from host running the installer/Ansible
+
+|*etcd Security Group*
+a| - tcp/2379 from Masters
+- tcp/2380 from etcd hosts
+
+|*Master Security Group*
+a| - tcp/8443 from 0.0.0.0/0
+- tcp/53 from All OpenShift hosts
+- udp/53 from All OpenShift hosts
+
+|*Node Security Group*
+a| - tcp/10250 from Masters
+- tcp/4789 from Nodes
+
+|*Infrastructure Nodes*
+(ones that can host the `*openshift-router*`)
+a|- tcp/443 from 0.0.0.0/0
+- tcp/80 from 0.0.0.0/0
+
+|===
+
+If configuring ELBs for load balancing the Masters and/or routers, you also need
+to configure Ingress and Egress security groups for the ELBs appropriately.
+
+*Override Detected IP Addresses and Host Names*
+
+Some deployments will require that the user override the detected host names and
+IP addresses for the hosts. To see what the default values will be, run the
+`*openshift_facts*` playbook:
+
+====
+----
+ansible-playbook playbooks/byo/openshift_facts.yml
+----
+====
+
+Now, verify the detected common settings. If they are not what you expect them
+to be, you can override them.
+
+[cols="1,2"]
+|===
+|`*hostname*`
+a| - Should resolve to the internal IP from the instances themselves.
+- `*openshift_hostname*` will override.
+
+|`*ip*`
+a| - Should be the internal IP of the instance.
+- `*openshift_ip*` will override.
+
+|`*public_hostname*`
+a| - Should resolve to the external IP from hosts outside of the cloud.
+- Provider openshift_public_hostname will override.
+
+|`*public_ip*`
+a| - Should be the externally accessible IP associated with the instance.
+- `*openshift_public_ip*` will override.
+
+|`*use_openshift_sdn*`
+a| - Should be true unless the cloud is GCE.
+- `*openshift_use_openshift_sdn*` overrides.
+
+|===
+
+If setting `*openshift_hostname*` to something other than the metadata-provided
+`*private-dns-name*` value, the native cloud integration for those providers
+will no longer work.
+
+In AWS, here are reasons why a user should override the variables:
+
+[cols="1,2"]
+|===
+|`*hostname*`
+a|The user is installing in a VPC that is not configured for both `*DNS hostnames*` and `*DNS resolution*`.
+
+|`*ip*`
+a|Possibly if they have multiple network interfaces configured and they want to
+use one other than the default. However, the support for this would be dependent on
+setting `*openshift_node_set_node_ip*`` to `True`. Otherwise, the SDN would attempt to
```

Comment:
The "However, ... would be ..." sounds awkward to my ears.  Suggest: "You must first set ____ to True."